app/controllers/follows_controller.rb

```diff
@@ -25,6 +25,28 @@ def show
     end
   end
 
+  def bulk_show
+    skip_authorization
+
+    response = params[:ids].map(&:to_i).each_with_object({}) do |id, hsh|
```

Comment:
this results in a 500 error if the client has no ids, we should use the classic `params.require(:ids)` check, so the client gets a 400 bad request.

Also:

```ruby
> "a".to_i
0
```

which means that it does stuff with user id that don't exist.

Reply:
Also we might want to avoid triggering behavior for user ids that don't exist as well